src/worklets/shell.cljs

```diff
@@ -0,0 +1,51 @@
+(ns worklets.shell)
+
+(def shell-worklets (js/require "../src/worklets/js/shell.js"))
+
+(defn stack-opacity
+  [id selected-stack-id]
+  (.stackOpacity ^js shell-worklets id selected-stack-id))
+
+(defn stack-pointer
+  [id selected-stack-id]
+  (.stackPointer ^js shell-worklets id selected-stack-id))
+
+(defn bottom-tabs-height
+  [home-stack-state-sv container-height extended-container-height]
+  (.bottomTabsHeight ^js shell-worklets home-stack-state-sv container-height extended-container-height))
+
+(defn bottom-tab-icon-color
+  [id selected-stack-id-sv home-stack-state-sv pass-through-sv color1 color2 color3]
+  (.bottomTabIconColor ^js shell-worklets
+                       id
+                       selected-stack-id-sv
+                       home-stack-state-sv
+                       pass-through-sv
+                       color1
+                       color2
+                       color3))
```

Comment:
Better to keep their names? (selected-tab-color, default-color, pass-through-color)

Reply:
ok